kong/plugins/datadog/handler.lua

```diff
@@ -25,59 +25,52 @@ local get_consumer_id = {
   end,
   username    = function(consumer)
     return consumer and consumer.username
-  end
+  end,
```

Comment:
Avoid unnecessary changes in the patch..

Reply:
@hbagdi This was in the style guide:
https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#tables

I added the trailing space.